autotst/conformer/systematic.py

```diff
@@ -40,16 +40,24 @@
 from ase import calculators
 from ase.calculators.calculator import FileIOCalculator
 from ase.optimize import BFGS
+from ase import units
+from ase.constraints import FixBondLengths
 
 import autotst
 from autotst.species import Conformer
 from autotst.reaction import TS
 from autotst.conformer.utilities import get_energy, find_terminal_torsions
 
+from rmgpy.molecule import Molecule
+
```

Comment:
Minor formatting thing, but group the `rmgpy` imports together and have them come after the `rdkit` imports. And make both of these imports come before `autotst` imports.